### PR TITLE
Papa 7a glider cals

### PR DESCRIPTION
**Pull request by cdobs**
Papa 7a glider cals

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09288__20190329.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9288,CC_calibration_date,1553817600000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09288__20190329.csv:2]: looks good

File: calibration/CTDGVM/CGINS-CTDGVM-00104__20200322.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+104,CC_calibration_date,1584835200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-00104__20200322.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00341__20200305.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+341,CC_calibration_date,1583366400000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00341__20200305.csv:2]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00537__20200211.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+537,CC_calibration_date,1581379200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/DOSTAM/CGINS-DOSTAM-00537__20200211.csv:2]: looks good

File: calibration/FLORDM/CGINS-FLORDM-03063__20200429.csv [hunk 5]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3063,CC_angular_resolution,1.096,
+3063,CC_depolarization_ratio,0.039,
+3063,CC_measurement_wavelength,700.0,
+3063,CC_scattering_angle,140.0,
```
Comment (rtravis-whoi) [inline calibration/FLORDM/CGINS-FLORDM-03063__20200429.csv:5]: looks good

File: calibration/FLORTM/CGINS-FLORTM-04039__20190326.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4039,CC_angular_resolution,1.076,
+4039,CC_depolarization_ratio,0.039,
+4039,CC_measurement_wavelength,700,
+4039,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTM/CGINS-FLORTM-04039__20190326.csv:5]: looks good

File: calibration/FLORTO/CGINS-FLORTO-01333__20190327.csv [hunk 7]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1333,CC_angular_resolution,1.076,
+1333,CC_depolarization_ratio,0.039,
+1333,CC_measurement_wavelength,700,
+1333,CC_scattering_angle,124,
```
Comment (rtravis-whoi) [inline calibration/FLORTO/CGINS-FLORTO-01333__20190327.csv:5]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20200106.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
... 271.26, 272.07, 272.88, 273.69, 274.5, 275.32, 276.13, 276.94, 277.75, 278.56, 279.38, 280.19, 281, 281.81, 282.62, 283.44, 284.25, 285.06, 285.87, 286.68, 287.5, 288.31, 289.12, 289.93, 290.75, 291.56, 292.37, 293.18, 293.99, 294.81, 295.62, 296.43, 297.24, 298.05, 298.87, 299.68, 300.49, 301.3, 302.11, 302.92, 303.74, 304.55, 305.36, 306.17, 306.98, 307.79, 308.61, 309.42, 310.23, 311.04, 311.85, 312.66, 313.47, 314.28, 315.1, 315.91, 316.72, 317.53, 318.34, 319.15, 319.96, 320.77, 321.58, 322.39, 323.2, 324.01, 324.82, 325.63, 326.44, 327.25, 328.06, 328.87, 329.68, 330.49, 331.3, 332.11, 332.92, 333.73, 334.54, 335.34, 336.15, 336.96, 337.77, 338.58, 339.39, 340.19, 341, 341.81, 342.62, 343.43, 344.23, 345.04, 345.85, 346.65, 347.46, 348.27, 349.07, 349.88, 350.69, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.13, 357.94, 358.74, 359.55, 360.35, 361.15, 361.96, 362.76, 363.57, 364.37, 365.17, 365.98, 366.78, 367.58, 368.38, 369.19, 369.99, 370.79, 371.59, 372.39, 373.2, 374, 374.8, 375.6, 376.4, 377.2, 378, 378.8, 379.6, 380.4, 381.2, 382, 382.8, 383.6, 384.39, 385.19, 385.99, 386.79, 387.59, 388.38, 389.18, 389.98, 390.77, 391.57, 392.37, 393.16, 393.96, 394.75]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00661__20200106.csv:8]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00350__20200915.csv [hunk 9]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+350,CC_cal_temp,20.00, source_file 3305-00327-00107_SN_350_Pre-Deployment_NUTNR-M.zip > SNA0350E.CAL
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00350__20200915.csv:2]: source_file should be 3305-00327-00107-A...

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
just need to add -A to source file for NUTNRM-00350 (that was my fault...)

File: calibration/NUTNRM/CGINS-NUTNRM-00350__20200915.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
...
...272.46, 273.26, 274.07, 274.87, 275.68, 276.49, 277.29, 278.1, 278.9, 279.71, 280.52, 281.32, 282.13, 282.93, 283.74, 284.55, 285.35, 286.16, 286.97, 287.77, 288.58, 289.39, 290.19, 291, 291.81, 292.62, 293.42, 294.23, 295.04, 295.85, 296.65, 297.46, 298.27, 299.08, 299.88, 300.69, 301.5, 302.31, 303.12, 303.92, 304.73, 305.54, 306.35, 307.16, 307.96, 308.77, 309.58, 310.39, 311.2, 312.01, 312.81, 313.62, 314.43, 315.24, 316.05, 316.86, 317.67, 318.47, 319.28, 320.09, 320.9, 321.71, 322.52, 323.33, 324.13, 324.94, 325.75, 326.56, 327.37, 328.18, 328.99, 329.79, 330.6, 331.41, 332.22, 333.03, 333.84, 334.64, 335.45, 336.26, 337.07, 337.88, 338.69, 339.49, 340.3, 341.11, 341.92, 342.73, 343.53, 344.34, 345.15, 345.96, 346.77, 347.57, 348.38, 349.19, 350, 350.8, 351.61, 352.42, 353.23, 354.03, 354.84, 355.65, 356.45, 357.26, 358.07, 358.87, 359.68, 360.49, 361.29, 362.1, 362.91, 363.71, 364.52, 365.32, 366.13, 366.94, 367.74, 368.55, 369.35, 370.16, 370.96, 371.77, 372.57, 373.38, 374.18, 374.99, 375.79, 376.6, 377.4, 378.21, 379.01, 379.81, 380.62, 381.42, 382.23, 383.03, 383.83, 384.64, 385.44, 386.24, 387.04, 387.85, 388.65, 389.45, 390.25, 391.06, 391.86, 392.66, 393.46, 394.26]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00350__20200915.csv:8]: looks good; fixed source_file name.

File: calibration/NUTNRM/CGINS-NUTNRM-00350__20200106.csv [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
...272.46, 273.26, 274.07, 274.87, 275.68, 276.49, 277.29, 278.1, 278.9, 279.71, 280.52, 281.32, 282.13, 282.93, 283.74, 284.55, 285.35, 286.16, 286.97, 287.77, 288.58, 289.39, 290.19, 291, 291.81, 292.62, 293.42, 294.23, 295.04, 295.85, 296.65, 297.46, 298.27, 299.08, 299.88, 300.69, 301.5, 302.31, 303.12, 303.92, 304.73, 305.54, 306.35, 307.16, 307.96, 308.77, 309.58, 310.39, 311.2, 312.01, 312.81, 313.62, 314.43, 315.24, 316.05, 316.86, 317.67, 318.47, 319.28, 320.09, 320.9, 321.71, 322.52, 323.33, 324.13, 324.94, 325.75, 326.56, 327.37, 328.18, 328.99, 329.79, 330.6, 331.41, 332.22, 333.03, 333.84, 334.64, 335.45, 336.26, 337.07, 337.88, 338.69, 339.49, 340.3, 341.11, 341.92, 342.73, 343.53, 344.34, 345.15, 345.96, 346.77, 347.57, 348.38, 349.19, 350, 350.8, 351.61, 352.42, 353.23, 354.03, 354.84, 355.65, 356.45, 357.26, 358.07, 358.87, 359.68, 360.49, 361.29, 362.1, 362.91, 363.71, 364.52, 365.32, 366.13, 366.94, 367.74, 368.55, 369.35, 370.16, 370.96, 371.77, 372.57, 373.38, 374.18, 374.99, 375.79, 376.6, 377.4, 378.21, 379.01, 379.81, 380.62, 381.42, 382.23, 383.03, 383.83, 384.64, 385.44, 386.24, 387.04, 387.85, 388.65, 389.45, 390.25, 391.06, 391.86, 392.66, 393.46, 394.26]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00350__20200106.csv:8]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20200106.csv (hunk 8 above)
Comment (snwhite) [reply calibration/NUTNRM/CGINS-NUTNRM-00661__20200106.csv:8]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00350__20200915.csv (hunk 10 above)
Comment (snwhite) [reply calibration/NUTNRM/CGINS-NUTNRM-00350__20200915.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]
All files look good.

**Pull request merged**